source/extensions/filters/http/rbac/rbac_filter.cc

```diff
@@ -97,6 +97,9 @@ Http::FilterHeadersStatus RoleBasedAccessControlFilter::decodeHeaders(Http::Head
 
       auto filter_meta = filter_metadata.at(HttpFilterNames::get().Rbac);
       filter_meta.MergeFrom(metrics);
+    } else {
```

Comment:
Not directly related, but can you add a test for the if statement true case?
https://84260-65214191-gh.circle-artifacts.com/0/build/envoy/generated/coverage/coverage.source_extensions_filters_http_rbac_rbac_filter.cc.html